quickstart/aspnet/README.md

```diff
@@ -18,23 +18,41 @@ This sample shows you how to use an ASP.NET web application to connect to Azure
 
 - Azure subscription - [create one for free](https://azure.microsoft.com/free/dotnet)
 - Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
-- [Visual Studio 2019](https://www.visualstudio.com/downloads/) with the **ASP.NET and web development** and **Azure development** workloads.
+- [Visual Studio](https://www.visualstudio.com/downloads/) with the **ASP.NET and web development** and **Azure development** workloads.
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 ## Set up the working environment
 
+### 1. Set up local credential for using Entra ID
+This sample uses Microsoft Entra ID for connecting to an Azure Cache for Redis instance.
+The following line of code in *ContosoTeamStats/RedisConnection.cs* obtains the default credential from your local machine or an Azure resource as the identity for authentication and authorization.
+
+```csharp
```

Comment:
Including this code snippet in the "Set up local credential for using Entra ID" is a little confusing, since people don't need to do anything with that code at this point. 
We should simply tell people to configure the Entra ID auth on their cache, and log in to VS or az CLI using an account that has permissions on the cache.

Reply:
fixed.